.travis.yml

```diff
@@ -1,6 +1,7 @@
 # We test each method for building LaTeX in a separate job.
 language: generic
 
+dist: xenian
```

Comment:
Looks like xenial (not xenian) is default since april this year, so I'd prefer not fixing the distro unless required
https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment